Gemfile.lock

```diff
@@ -829,4 +829,4 @@ DEPENDENCIES
   wisper-rspec
 
 BUNDLED WITH
-   2.5.6
+   2.4.22
```

Comment:
I think we're all on different versions of bundler? I've just done `bundle update --bundler` and I'm on 2.5.4.

Reply:
Have just updated to 2.5.6...